app/controllers/application_controller.rb

```diff
@@ -12,4 +12,16 @@ def current_user
   def authorize_user
     redirect_to login_path unless current_user
   end
+
+  def root_redirect
+    if current_user.admin?
+      admin_dashboard_path
+    elsif DemoNight.current && DemoNight.current.accepting_submissions?
+      new_project_path
+    elsif DemoNight.current && DemoNight.current.voting?
+      projects_path
```

Comment:
could we maybe combine this branch and the `DemoNight.current && DemoNight.current.accepting_submissions?` into a single branch + add another method that checks? I don't know.